### ISSUE TITLE
no scrollbar in Album List

### ISSUE THREAD

**Issue by jberkus** [voltraco/beta#10]
Version: Linux 64-bit, installed from tarball

In the "list of albums" view there is no scrollbar.  This means that any albums which don't fit in the first dozen or so are inaccessible for mouse-based browsing.  Holding down the CTRL key and using the arrow keys works, so scrolling is present, just not the scrollbar.  Scrollbar is present in Artists view.

Gnome 3, which probably matters.

**Comment by jberkus** [comment]
Update:  after pressing CTRL+DownArrow, the scrollbar appears.  It's not present until I do that, though.

**Comment by heapwolf** [comment]
The scrollbar we are using is meant to appear only when you use the mouse to scroll (and there are items to cause the scrollbar to appear). Sadly, there is no real keyboard support in the app to speak of. I'd love to tackle that before public beta starts.

**Comment by jberkus** [comment]
Hmm.  Well, if one doesn't have a mouse with a scrollwheel (which most laptops don't), then what?

**Comment by heapwolf** [comment]
Im not sure how common it is for a laptop to not have a pointer device (or the inability to attach one). But i'll admit, accessibility hasn't been a priority so far. We could certainly make a list of accessibility features.

**Comment by jberkus** [comment]
"pointer device"  != "scrollwheel"

There isn't any way by any amount of mouse-clicking to make the scrollbar appear.  Only CTRL+Arrow does it.

I think you're troubleshooting a different problem from the one I'm reporting.
1. Scroll to the top of your list of albums.
2. Close Voltra.
3. Open Voltra
4. Move the mouse to the right hand side of the window
5. No scrollbar appears
6. There is no way to scroll using the mouse pointer.

**Comment by heapwolf** [comment]
Personally, I haven't used a pointer device that doesnt have the ability to scroll in a very long time. Maybe you can tell me what kind of hardware set up you're on?

**Comment by jberkus** [comment]
http://www.logitech.com/en-us/product/trackman-marble (older version without the arrow buttons).

But there's also people who like pointing sticks, or folks who's software doesn't properly support two-finger scroll on touchpads, or have disabled it for some reason.  Regardless, "only works with scrollwheel" is a rather questionable design choice.

The problem is compounded because Voltra uses the arrow keys to control the menus, which means that just pressing an arrow key doesn't scroll either, the way it would in a browser.  You have to press CTRL, or whatever your desktop supports to simulate a scrollwheel.

Plus even people who _have_ a scrollwheel won't necessarily think to use it.  How many bug reports do you want to field on this, really?

**Comment by jberkus** [comment]
Oh, and please don't take away the arrow keys for menus.  That's a nice thing.

**Comment by jberkus** [comment]
The scrollbar is also missing for the list of tracks in your playlist.  Unlike the album list, *nothing* can make the scrollbar appear, not even pressing ctrl-arrow (although that does scroll).

**Comment by jberkus** [comment]
Just tested beta2.20, this issue is still present.

**Comment by jberkus** [comment]
Issue still present in 2.5.12
